.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,102 @@
+name: Tests / E2E
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - docs/**
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ibc-go-simd-e2e
+
+jobs:
+  docker-build:
+    if: ${{ !github.event.pull_request.head.repo.fork }}
```

Comment:
I won't be able to verify this until the PR is merged unfortunately. I'll keep an eye on the action and create a fix if there are any issues.